app/views/categories/new.html.erb

```diff
@@ -1,16 +1,20 @@
-<h1>Category Form</h1>
-
-<%= form_with(model:@category, url: categories_path, local: true) do |form| %>
-
-  <div class="field">
-    <%= form.label :name, "Category:" %>
-    <%= form.collection_select(:name, ["Food", "Clothes", "Health", "Rent", "School", "Savings", "Family", "Charity", "Investment", "Business", "Pet", "Maintenance", "Treats", "Transportation", "Other"], :to_s, :to_s, { prompt: "---" }, { class: '' }) %>
+<section class='form-category-container'>
+  <div class='form-category-content'>
+    <h1>Category Form</h1>
+    
+    <%= form_with(model:@category, url: categories_path, local: true, class: 'category-form') do |form| %>
+    
+      <div class="category-field">
+        <%= form.label :name, "Category" %>
+        <%= form.collection_select(:name, ["Food", "Clothes", "Health", "Rent", "School", "Savings", "Family", "Charity", "Investment", "Business", "Pet", "Maintenance", "Treats", "Transportation", "Other"], :to_s, :to_s, { prompt: "---" }, { class: '' }) %>
+      </div>
+    
+      <div class="category-field">
+      <%= form.label :icon, "Category Icon" %>
+      <%= form.collection_select(:icon, [ "🍔", "👚", "🏥", "🏫", "📚", "🐷", "👪", "🎁", "💌", "👔", "🐈", "🛠", "🏖", "🚌", "💡" ], :to_s, :to_s, { prompt: "---" }, { class: '' }) %>
+      </div>
+    
+        <div class="category-field submit-food-btn"><%= form.submit 'Submit Food', class: 'submit-food' %></div>
+    <% end %>
   </div>
```

Comment:
- [ ] In your Category form it would be nice to add a back button so if a user clicks on the "Back" button (\<), the user navigates to the home page, this will create a better user experience so when users don't want to create a category they can click the back button 🤗